### ISSUE TITLE
Missing variables.tf

### ISSUE THREAD

**Issue by mmagdy88** [hashicorp/learn-terraform-plan#1]
I would like to point out that "variables.tf" file is missing from this repo which is being used in the below tutorial:
https://learn.hashicorp.com/tutorials/terraform/plan?in=terraform/cli

By following the tutorial, we will manually create the file, but until then, it will show errors in
```
terraform plan
terraform plan -out tfplan
```

Because the variable is already in the main.tf without "variables.tf" file.

For beginners, following the tutorial step-by-step will get them stuck showing the following error:

```
│ Error: Reference to undeclared input variable
│
│   on main.tf line 34, in module "hello":
│   34:   secret_key = var.secret_key
│
│ An input variable with the name "secret_key" has not been declared. This variable can be declared with a variable "secret_key" {} block.
```

**Comment by mr-pf** [comment]
I just ran into this error.

**Comment by vitorstone** [comment]
Just delete `var.secret_key` on line 34 and write `"secret"` instead, to match what it says on the tutorial.

**Comment by spuder** [comment]
Here is the step later in the instructions where the user is to add a `variables.tf` and `terraform.tfvars` file

https://learn.hashicorp.com/tutorials/terraform/plan?in=terraform/cli#modify-configuration

**Comment by pipkin-insight** [comment]
The issue is that a 'plan' happens before the modification of the configuration. The work flow appears to be init > plan > modify > plan. If the first plan doesn't work, then you have people trying to figure out what is broken.

I agree with the proposed change. The tutorial is broken in workflow. I don't have a disagreement that one ought to know how to resolve the issue at this point; though there ought not be an issue with the workflow.

**Comment by robin-norwood** [comment]
Sorry for the delay in responding to this! Fixed by #11
